app/src/main/java/com/evelynhasama/bucketbuds/FriendsAdapter.java

```diff
@@ -169,7 +166,7 @@ public void updateFriends(Boolean added, int position, UserPub otherUserPub){
             public void done(ParseException e) {
                 if (e != null) {
                     Toast.makeText(context, "Friend action failed", Toast.LENGTH_SHORT).show();
-                    Log.e(TAG, String.valueOf(e));
+                    Log.d(TAG, String.valueOf(e));
```

Comment:
nit: Why `Log.d` instead of `Log.e`?
Is the exception expected in some scenarios?

IF the exception is truly an exception (you don't expect it to happen), then `Log.e` is correct.

If there are _some_ exceptions that are expected, then you can use different `Log.e/Log.w/Log.i` depending on what you expect or not.

Reply:
I got really used to using Log.d for debugging and decided to use them for everything, but yeah I should use the appropriate Log.